ember serve, ember deploy fail with ember-cli 0.2.0

I just ran `npm install ember-cli-deploy --save-dev`.

When trying to run `ember serve` or `ember deploy` with `ember-cli-deploy 0.4.0`, ember complains about the following:

```
Cannot find module 'glob'
Error: Cannot find module 'glob'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/Users/walti/Projects/distillery/node_modules/ember-cli-deploy/lib/utilities/detect-deprecated-config.js:2:12)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
```

It seems that the `glob` dependency is not installing automatically. If I do a `npm install --save-dev glob` in my ember directory, it then works.

How does one fix this?

--- a/package.json
+++ b/package.json
@@ -34,7 +34,6 @@
     "ember-export-application-global": "^1.0.0",
     "express": "^4.8.5",
     "github": "0.2.3",
-    "glob": "^4.0.5",
     "mocha": "^2.0.1",
     "sinon": "^1.12.1"
   },
@@ -52,6 +51,7 @@
     "chalk": "^0.5.1",
     "core-object": "0.0.2",
     "execSync": "^1.0.2",
+    "glob": "^4.0.5",
     "lodash-node": "^2.4.1",
     "mkdirp": "^0.5.0",
     "ncp": "^1.0.1",

